<?xml version="1.0"?>
<robot name="ur5">
    <group_state name="home" group="manipulator">
        <joint name="elbow_joint" value="2.74894851237" />
        <joint name="shoulder_pan_joint" value="-1.57086762826" />
        <joint name="wrist_3_joint" value="-6.42953709757e-05" />
        <joint name="wrist_1_joint" value="1.57081943395" />
        <joint name="shoulder_lift_joint" value="-1.74816935325" />
        <joint name="wrist_2_joint" value="-1.57072034365" />
    </group_state>
    <group_state name="handoff" group="manipulator">
        <joint name="elbow_joint" value="-2.30076401719" />
        <joint name="shoulder_pan_joint" value="0.610577442791" />
        <joint name="wrist_3_joint" value="-2.91388231902" />
        <joint name="wrist_1_joint" value="-2.35121686528" />
        <joint name="shoulder_lift_joint" value="-1.35215015628" />
        <joint name="wrist_2_joint" value="-2.51391633578" />
    </group_state>
    <group_state name="look_at_bin" group="manipulator">
        <joint name="elbow_joint" value="-1.86989988775" />
        <joint name="shoulder_pan_joint" value="2.31428162707" />
        <joint name="wrist_3_joint" value="0.741777603206" />
        <joint name="wrist_1_joint" value="-1.64583086855" />
        <joint name="shoulder_lift_joint" value="-1.19656687163" />
        <joint name="wrist_2_joint" value="1.57081836557" />
    </group_state>
    <group_state name="dope_to_rviz" group="manipulator">
        <joint name="elbow_joint" value="1.92118022798" />
        <joint name="shoulder_pan_joint" value="2.16366122517" />
        <joint name="wrist_3_joint" value="-2.88802868637" />
        <joint name="wrist_1_joint" value="-0.515408719901" />
        <joint name="shoulder_lift_joint" value="-1.8783918195" />
        <joint name="wrist_2_joint" value="-0.959934662872" />
    </group_state>
</robot>